addons/web/static/src/public/caps_lock_warning.js

```diff
@@ -0,0 +1,32 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+export class CapsLockWarning extends Interaction {
+    static selector = ".caps-lock-warning";
```

Comment:
In general, our classes are rather using `o_` and underscore... not a fan but that our convention, to distinguish from bootstrap and others. So for new classes, try to use that (same for all new classes).